src/app/qgsdxfexportdialog.cpp

```diff
@@ -797,6 +809,283 @@ void QgsDxfExportDialog::deselectDataDefinedBlocks()
 }
 
 
+void QgsDxfExportDialog::loadSettingsFromFile()
+{
+  const QString fileName = QFileDialog::getOpenFileName( this, tr( "Load DXF Export settings" ),
+                           QgsDxfExportDialog::settingsDxfLastSettingsDir->value(),
+                           tr( "XML file" ) + " (*.xml)" );
+  if ( fileName.isNull() )
+  {
+    return;
+  }
+
+  bool resultFlag = false;
+
+  QDomDocument domDocument( QStringLiteral( "qgis" ) );
+
+  // location of problem associated with errorMsg
+  int line, column;
+  QString errorMessage;
+
+  QFile file( fileName );
+  if ( file.open( QFile::ReadOnly ) )
+  {
+    QgsDebugMsgLevel( QStringLiteral( "file found %1" ).arg( fileName ), 2 );
+    // read file
+    resultFlag = domDocument.setContent( &file, &errorMessage, &line, &column );
+    if ( !resultFlag )
+      errorMessage = tr( "%1 at line %2 column %3" ).arg( errorMessage ).arg( line ).arg( column );
+    file.close();
+  }
+
+  if ( QMessageBox::question( this,
+                              tr( "DXF Export load from XML file" ),
```

Comment:
```suggestion
                              tr( "DXF Export - Load from XML File" ),
```
title case and split the title (assuming we need the "dxf export" bits). Otherwise I am afraid for translations